src/shared/image-policy.c

```diff
@@ -445,6 +459,21 @@ int partition_policy_flags_to_string(PartitionPolicyFlags flags, bool simplify,
                         l[m++] = "growfs-on";
         }
 
+        if (flags & PARTITION_POLICY_BTRFS)
```

Comment:
please add a comment here, clarifying that the flag names returned here shall be the literal fstype names that we can pass to /bin/mount's -t switch, and that we rely on this.

(see other comment)